android/modules/ui/src/java/ti/modules/titanium/ui/ShortcutModule.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2020 by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+package ti.modules.titanium.ui;
+
+import org.appcelerator.kroll.KrollModule;
+import org.appcelerator.kroll.annotations.Kroll;
+
+/**
+ * This module exist so `Ti.UI.Shortcut.addEventListener()` is defined.
+ * `Ti.UI.createShortcut` is manually defined in `UIModule`.
+ */
+@Kroll.module(parentModule = UIModule.class)
+public class ShortcutModule extends KrollModule
```

Comment:
Feels a bit odd to have this module stub so we can hang event listeners. Looks like we also always fire the event at the same time to Ti.App `shortcutitemclick` event - so you could probably hack this pretty easily in a js extension (basically add a dumb Ti.UI.Shortcut add/removeEventListener that registers for Ti.App event and forwards it)